WcaOnRails/app/views/competitions/_index_competitions_list.html.erb

```diff
@@ -50,4 +43,35 @@
       });
     });
   </script>
+<% else %>
+  <% if current_user&.can_see_admin_competitions? %>
+    <% if competitions.empty? %>
+      <%= render 'no_competitions_found' %>
+    <% else %>
+      <% past_competitions, upcoming_competitions = competitions.partition(&:is_over?) %>
```

Comment:
Stumbled upon this and call it present competitions on the comps index but in the code we can have more descriptive name =)